pumpkin-core/src/math/boundingbox.rs

```diff
@@ -33,10 +50,25 @@ impl BoundingBox {
         }
     }
 
+    pub fn bounding_boxes_intersect(&self, other: &BoundingBox) -> bool {
```

Comment:
using the struct/modules name in functions is a bad design, You should always keep it simple, Also its wrong, You wrote intersect. Just name it `intersects`

Reply:
oh bruh i actually know, dont know why i did that, lemme fix